### PR TITLE
feat: make actions with confirmation page possible

### PR DESCRIPTION
**Pull request by chrfritsch**
- Switched to media because there are more default actions
- Delete actions has a confirmation page

### CODE REVIEW

File: src/Form/ActionForm.php [hunk 1]
```diff
@@ -3,18 +3,22 @@
...
+use Drupal\Core\Url;
 use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
 
 /**
  * Provides the form functions to call actions on referenced entities.
  */
 class ActionForm implements ContainerInjectionInterface {
 
+  use DependencySerializationTrait;
```
Comment (ol0lll) [inline src/Form/ActionForm.php:21]: Whut?

File: src/Form/ActionForm.php [hunk 2]
```diff
@@ -174,6 +188,26 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       });
 
       $action->execute($entities);
+
+      $operation_definition = $action->getPluginDefinition();
+      if (!empty($operation_definition['confirm_form_route_name'])) {
+        $url = Url::fromUserInput($this->request->getPathInfo(), ['query' => $this->request->query->all()]);
+        $options = [
+          'query' => ['destination' => $url->toString()],
```
Comment (ol0lll) [inline src/Form/ActionForm.php:196]: Should test this.

Comment (ol0lll) [reply src/Form/ActionForm.php:196]: follow-up?

**Review by ol0lll — APPROVED** [review]

File: src/Form/ActionForm.php (hunk 2 above)
Comment (chrfritsch) [reply src/Form/ActionForm.php:196]: Opened #5

**Pull request merged**